src/holodeck/weather.py

```diff
@@ -37,7 +37,7 @@ def set_day_time(self, hour):
         By the next tick, the lighting and the skysphere will be updated with the new hour.
 
         If there is no skysphere, skylight, or directional source light in the world, this command
```

Comment:
OK, I know you already changed this once, but thinking about it, aren't we expecting every world to have these features? So is this line really needed (ie it is expected that every world we provide will have this, so we don't need this message)

Reply:
I think I agree--@MaxRobinsonTheGreat what's your take?

Reply:
For world development documentation I think it is useful to keep.